File: doc/codes/python/examples/api/compute/modulation/sine-example.py
from autd3.units import Hz, rad
from autd3.value import SamplingConfig
from autd3_modulation import SineOption, modulation_buffer, sine

out = modulation_buffer()

sine(
    150 * Hz,
    SineOption(
        amplitude=0xFF,
        offset=0x80,
        phase=0.0 * rad,
        clamp=False,
        sampling_config=SamplingConfig.FREQ_4K,
    ),
    out,
)

File: doc/codes/csharp/examples/api/compute/modulation/sine-example.cs
using AUTD3;
using static AUTD3.Units;

// HIDE
namespace DocSamples.ApiComputeModulationSineExample;

internal static class Sample
{
    internal static void Run()
    {
        // HIDE_END
var dst = Modulation.ModulationBuffer();

Modulation.Sine(
    150 * Hz,
    new SineOption
    {
        Amplitude = 0xFF,
        Offset = 0x80,
        Phase = 0.0f * rad,
        Clamp = false,
        SamplingConfig = SamplingConfig.Freq4k,
    },
    dst
);
        // HIDE
    }
}
// HIDE_END
